src/site/content/en/blog/css-module-scripts/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Using CSS Module Scripts to import stylesheets
+subhead: >
+  Learn how to use CSS module scripts to import CSS stylesheets using the same syntax as JavaScript
+  modules.
+description: >
+  Learn how to use CSS module scripts to import CSS stylesheets using the same syntax as JavaScript
+  modules.
+authors:
+  - dandclark
+date: 2021-08-06 # TODO Update to publishing date
+hero: image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/Hu0ljomDqgb0y7h7REp6.jpg
+alt: A stack of multicolored shipping containers.
+tags:
+  - blog
+  - css
+  - DOM
+  - javascript
+  - modules
+---
+
+With the new CSS module scripts feature, you can load CSS style sheets with `import` statements,
+just like [JavaScript
+modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules). The style sheets
+can then be applied to documents or [shadow
+roots](https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot) in the same manner as
+[constructable
+stylesheets](https://developers.google.com/web/updates/2019/02/constructable-stylesheets). This can
+be more convenient and [more
+performant](https://dandclark.github.io/json-css-module-notes/#css-module-performancememory-examples)
+than other ways of importing and applying CSS.
+
+## Browser Support
+
+CSS module scripts are available in Chrome and Edge in version 93. 
```

Comment:
something about unflagged? A lot of stuff here goes out being flagged, so it might be worth calling that out.

Reply:
Changed this to specify that it's available by default in 93.